test/helpers/factory.js

```diff
@@ -18,11 +19,11 @@ export function gistData({html, css, javascript, enabledLibraries = []} = {}) {
       content: javascript,
     });
   }
-  if (!isEmpty(enabledLibraries)) {
```

Comment:
Also please let me know if my comment doesn't make sense, I'm running on fumes at this point and wouldn't be surprised if I'm incomprehensible.